setup.py

```diff
@@ -0,0 +1,34 @@
+import codecs
+import os
+from setuptools import setup, find_packages
+
+here = os.path.abspath(os.path.dirname(__file__))
+
+with codecs.open(os.path.join(here, 'README.rst'), encoding='utf-8') as f:
+    README = f.read()
```

Comment:
if you're going to read this from your setup.py you have to also add `README.rst` to [MANIFEST.in](https://docs.python.org/2/distutils/sourcedist.html#the-manifest-in-template)